### PR TITLE
Adds Drink: Pumpkin Spice Latte

### PR DESCRIPTION
**Pull request by t-toasted**
## About the PR 
Adds in a pumpkin spice latte recipe to the game. Base recipe: pumpkin juice, coffee, and milk. Also, consequently, adds in pumpkin juice to the game; juice a pumpkin today!

## Why's this needed? 
Pumpkins don't have a lot of uses currently. This gives them a common, everyday use that works with the upcoming season. Gives bartenders another drink to make. Also lays the groundwork for a future pumpkin mutation. 

## Changelog

```changelog
(u)Roddy
(+)Added pumpkin juice and pumpkin spice lattes (milk, pumpkin juice, and coffee)
```

### CODE REVIEW

**Comment by DimWhat** [comment]
you should probably explain the recipe in the changelog. Be sure to include that you added pumpkin juice

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -4398,3 +4410,16 @@ datum
...
+		fooddrink/pumpkinspicelatte
+			name = "pumpkin spice latte"
+			id = "pumpkinspicelatte"
+			id = "pumpkinspicelatte"
+			fluid_r = 231
+			fluid_g = 106
+			fluid_b = 0
+			description = "Whether or not it contains actual pumpkin juice has been up for debate."
+			reagent_state = LIQUID
+			taste = "earthy and sweet"
```
Comment (TobleroneSwordfish) [inline code/modules/chemistry/Reagents-FoodDrink.dm:4423]:
```suggestion
			taste = list("earthy", "sweet")
```
I think this is meant to be like this?

**Comment by t-toasted** [comment]
!mergeupstream

**Comment by flrsh** [comment]
Changelogs should be in past tense. You can probably also condense this into one entry `(+)Added pumpkin juice and pumpkin spice lattes (milk, pumpkin juice, and coffee)`

**Comment by t-toasted** [comment]
> Changelogs should be in past tense. You can probably also condense this into one entry `(+)Added pumpkin juice and pumpkin spice lattes (milk, pumpkin juice, and coffee)`

Donezo bunzo

**Pull request merged**